calicoctl/commands/delete.go

```diff
@@ -25,6 +25,9 @@ import (
 func Delete(args []string) error {
 	doc := EtcdIntro + `Delete a resource identified by file, stdin or resource type and name.
 
+Valid resource kinds are bgpPeer, hostEndpoint, policy, pool, profile and tier.  The <KIND>
```

Comment:
Looks like they do all lowercase mushed together. I guess making them camel-case but case-insensitive means its just a tad easier to read, so I'm fine with it.